src/risedevtool/src/bin/risedev-dev.rs

```diff
@@ -323,7 +323,10 @@ fn task_main(
                         risedev::TcpReadyCheckTask::new(c.address.clone(), c.port, c.user_managed)?;
                     task.execute(&mut ctx)?;
                 } else {
-                    let mut task = risedev::LogReadyCheckTask::new("Ready for start up.")?;
+                    // When starting a MySQL container, the MySQL process is set as the main process.
+                    // Since the first process in a container always gets PID 1, the MySQL log always shows
+                    // "starting as process 1".
+                    let mut task = risedev::LogReadyCheckTask::new("starting as process 1")?;
```

Comment:
I'm not sure if this line indicates that the server is truly ready.

What about using `ready for connections` instead?
![telegram-cloud-photo-size-1-5158891378112048407-y](https://github.com/risingwavelabs/risingwave/assets/25862682/c6efbd63-149b-46e3-8ade-2ea1bade796c)

Reply:
When MySQL is launched for the first time, it will restart once initialization is complete. The log will contains two `ready for connections`:
![截屏2024-06-20 14 28 00](https://github.com/risingwavelabs/risingwave/assets/14160025/d2c84e09-c976-4de6-b175-dcffce47c2b5)

But I also found that the log might contains `starting as process 125` as above. 🥵 How about change the pattern to `ready for connections` and add a sleep here? We need a more robust mechanism for health checks in the future.

Reply:
This is indeed a problem. 😕 If it still doesn't work well then we may switch to introduce dependency on `mysql` and try to connect in `risedev-dev`.